pkgs/applications/editors/emacs-modes/melpa-packages.nix

```diff
@@ -116,6 +116,13 @@ let
           stripDebugList = [ "share" ];
         });
 
+        erlang = super.erlang.overrideAttrs (attrs: {
+          buildInputs = attrs.buildInputs ++ [
+            pkgs.perl530
```

Comment:
Does this not work with a more recent perl?

Reply:
That's a good question.  The configure script of erlang seems to suggest 5 and I thought `pkgs.perl` and `pkgs.perl530` point to the same thing.  I'll try building with a different version of perl and update if it works with a new version.

Reply:
I see now, `pkgs.perl` and `pkgs.perl530` are clearly not the same.  I'll forcepush a change to simply `pkgs.perl`.

Reply:
Great! Thank you.